_benchmark/workloads/vectorsearch.md

```diff
@@ -0,0 +1,572 @@
+| `query_max_distance `                     | The maximum distance to be returned for the vector search. Only one of `query_k`, `query_max_distance`, or `query_min_score` can be provided. |
+| `query_min_score`                         | The minimum score to be returned for the vector search. Only one of `query_k`, `query_max_distance`, or `query_min_score` can be provided.    |
+| `query_data_set_format`                   | The format of the vector dataset used for queries. Only one of `query_k`, `query_max_distance`, or `query_min_score` can be provided.        |
+| `query_data_set_path`                     | The path to the vector dataset used for queries.                                                                                               |
+| `query_count`                             | The number of queries used for the search operation.                                                                                                 |
+| `query_body`                              | The JSON properties that will be merged with the search body.                                                                                        |
+| `search_clients`                          | The number of clients used to run queries.                                                                                               |
+| `repetitions`                             | The number of repetitions completed until the dataset is exhausted. Default is `1`.                                                                      |
+| `target_throughput`                       | The target throughput for each query operation, in requests per second. Default is `10`.                                                         |
+| `time_period`                             | The period of time dedicated to running the benchmark test, in seconds. Default is `900`.                                                       |
+
+
+
+## Test procedures
+
+The workload supports the following test procedures.
+
+### No train test
+
+The no train test procedure tests vector search indexes that require no training. You can define the underlying configuration of the vector search algorithm (such as specifying a specific engine or space type) as method definitions. 
+
+### No train test (index only)
+
+The no train test (index only) procedure is used to index vector search indexes that require no training. This can be particularly useful when you want to benchmark only the indexing operation.
+
+### No train test (AOSS)
+
+The no train test (AOSS) procedure is similar to the no train test procedure but is specifically used for Amazon OpenSearch Serverless vector search collections. This procedure doesn't include operations like **refresh** and **warmup** because they aren't supported by vector search collections.
```

Comment:
```suggestion
The no train test procedure for Amazon OpenSearch Serviceless procedure is used specifically for Amazon OpenSearch Serverless vector search collections. This procedure doesn't include operations like **refresh** and **warmup** because they aren't supported by vector search collections.
```